utils/helpers.py

```diff
@@ -89,3 +97,33 @@ def get_csv_from_url(url, filepath):
 
     print('The url does not point to a valid a csv file')
     return None
+
+
+def record_skipped(record, file_path):
+    """
+    Get csv from URL
+    :param record: record  that was skipped
+    :param file_path: path to the output file
+    :return: None
+    """
+    filename = "/skipped.txt"
```

Comment:
`the file should have the same columns as the one processed` - the file should be csv to allow the user to edit and reimport  :)